Reference/Searching/Examine/Indexing/index.md

```diff
@@ -22,212 +13,123 @@ You can modify the built in indexes in several ways:
-    {
-    }
-
-    // note - in Umbraco 8.4 this method is obsoleted and it's advised to replace the
-    // IUmbracoIndexConfig as in the above example.
-    public override IValueSetValidator GetMemberValueSetValidator()
+    public class ConfigureExamineOptions : IConfigureNamedOptions<LuceneDirectoryIndexOptions>
     {
-        // all built in member properties to exclude (i.e. Password question, etc...)
-        var excludeFields = Constants.Conventions.Member.GetStandardPropertyTypeStubs().Keys;
+        public void Configure(string name, LuceneDirectoryIndexOptions options)
+        {
+            if (name.Equals(Constants.UmbracoIndexes.ExternalIndexName))
+            {
+                options.FieldDefinitions.AddOrUpdate(new FieldDefinition("price", FieldDefinitionTypes.Double));
+            }
+        }
 
-        // include everything except the above
-        return new MemberValueSetValidator(null, null, null, excludeFields);
+        // Part of the interface, but does not need to be implemented for this.
+        public void Configure(LuceneDirectoryIndexOptions options)
+        {
+            throw new System.NotImplementedException();
+        }
     }
 }
 ```
+This will either add the `price` field to the index, or if the field already exists, update date its type to double.
 
-Another example could be to replace the default Lucene analyzer for the ExternalIndex:
-
-```c#
-using System.Collections.Generic;
-using Examine;
-using Lucene.Net.Analysis;
-using Umbraco.Core;
-using Umbraco.Core.Logging;
-using Umbraco.Core.Services;
-using Umbraco.Examine;
-using Umbraco.Web.Search;
-
-// override the default creator
-public class CustomUmbracoIndexesCreator : UmbracoIndexesCreator
-{
-    // ctor
-    public CustomUmbracoIndexesCreator(IProfilingLogger profilingLogger, ILocalizationService languageService, IPublicAccessService publicAccessService, IMemberService memberService, IUmbracoIndexConfig umbracoIndexConfig)
-        : base(profilingLogger, languageService, publicAccessService, memberService, umbracoIndexConfig)
-    {
-    }
+## Changing IValueSetValidator
 
-    // override the Create method
-    public override IEnumerable<IIndex> Create()
-    {
-        // create the default index definitions
-        var defaultIndexes = base.Create().ToDictionary(x => x.Name, x => x);
-        var internalIndex = defaultIndexes[Constants.UmbracoIndexes.InternalIndexName];
-        var memberIndex = defaultIndexes[Constants.UmbracoIndexes.MembersIndexName];
+An `IValueSetValidator` is responsible for validating a `ValueSet` to see if it should be included in the index and/or filtering the data in the `ValueSet`. For example, by default the validation process for the ExternalIndex checks if a `ValueSet` has a category type of either "media" or "content" (not member). If a `ValueSet` was passed to the ExternalIndex and it did not pass this requirement it would be ignored.
```

Comment:
I don't think people will understand the "filtering the data" part of this phrase. Perhaps this can be clarified some?

Reply:
@kjac I've tried adding an explanation for filtering 👍

Reply:
@Zeegaan much better! Thank you 😄